src/solvers/adaptive_time_solver.C

```diff
@@ -121,7 +136,28 @@ void AdaptiveTimeSolver::advance_timestep ()
      _system.get_dof_map().get_send_list());
 }
 
+void AdaptiveTimeSolver::adjoint_advance_timestep ()
+{
+  // Call the core time solver's adjoint_advance_timestep
+  core_time_solver->adjoint_advance_timestep();
```

Comment:
This makes the lack of a shim to ```core_time_solver->advance_timestep()``` above even more surprising; surely the operations here should be more symmetric?